WordPress/gutenberg
# [Mobile] - Move Appium drivers setup to bash script

This is part of the Appium 2 Migration https://github.com/WordPress/gutenberg/pull/55166

## What?
This PR moves the `appium:setup` inline script to a bash file. It also renames it to `test:e2e:setup`

## Why?
To reduce the amount of inline scripting we currently have in the `package.json` as well as making it easier to read/modify this file in the future.

## How?
Moves the content of the script to a new bash file, it also takes into account if the `appium` command exists or not, if not it will look for it from the `node_modules` folder, this is the case if you trigger the script file directly instead of triggering from a `package.json` script.

## Testing Instructions
- CI checks should pass.
- Running `npm run native test:e2e:setup` should work.
- Running this script directly with `./test-e2e-setup.sh` should work as well.

### Testing Instructions for Keyboard
N/A

## Screenshots or screencast 
N/A
